src/cascadia/TerminalSettingsEditor/ProfileViewModel.h

```diff
@@ -51,9 +51,52 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             Opacity(static_cast<float>(value) / 100.0f);
         };
 
-        void SetPadding(double value)
+        void SetLeftPadding(double value)
         {
-            Padding(to_hstring(value));
+            const hstring& padding = _GetNewPadding(PaddingDirection::Left, value);
+
+            Padding(padding);

```

Comment:
Oh, I understand now. This will properly propagate the change notification through to Padding. You may still need to do the other notifications yourself.

Reply:
That will allow you to turn this into a setter and a getter with normal two-way binding.